code/game/gamemodes/gang/gang.dm

```diff
@@ -28,16 +28,27 @@ GLOBAL_LIST_INIT(gang_colors_pool, list("red","orange","yellow","green","blue","
 	required_enemies = 2
 	recommended_enemies = 2
 	enemy_minimum_age = 14
+	report = TRUE
 
 	announce_span = "danger"
 	announce_text = "A violent turf war has erupted on the station!\n\
 	<span class='danger'>Gangsters</span>: Take over the station with a dominator.\n\
 	<span class='notice'>Crew</span>: Prevent the gangs from expanding and initiating takeover."
 
+	var/area/ai_monitored/security/armory/target_armory
+
 ///////////////////////////////////////////////////////////////////////////////
 //Gets the round setup, cancelling if there's not enough players at the start//
 ///////////////////////////////////////////////////////////////////////////////
 /datum/game_mode/gang/pre_setup()
+	for(var/area/ai_monitored/security/armory/A in GLOB.sortedAreas)
+		if(A.z == ZLEVEL_STATION)
+			target_armory = A
```

Comment:
I don't personally see a reason in out-right removing any chance of a loyalist group to compete against gangs. If the armory and equipment is getting bombarded round-start, it becomes a lot harder to mass capture/implant.